app/pb_kits/playbook/pb_legend/_legend.jsx

```diff
@@ -38,24 +36,23 @@ const Legend = (props: LegendProps) => {
 
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
-  const darkClass = dark ? 'dark' : 'light'
-  const bodyCSS = classnames(
-    buildCss('pb_legend_kit', color, darkClass), className,
+  const bodyCss = classnames(
+    buildCss('pb_legend_kit', color), className,
     globalProps(props)
   )
+  const bodyColor = bodyCss.includes('dark') ? 'lighter' : 'light'
 
   return (
     <div
         {...ariaProps}
         {...dataProps}
-        className={bodyCSS}
+        className={bodyCss}
         id={id}
     >
-      <Body color={dark ? 'lighter' : 'light'}>
+      <Body color={bodyColor}>
         <span className="pb_legend_indicator_circle" />
         <If condition={prefixText}>
           <Title
```

Comment:
Can we just pass the dark prop to the title kit instead?

Reply:
I am happy to do it this way, but I want to make sure everyone is on the same page. What do you think @thestephenmarshall and @jasperfurniss? I know we discussed this last week and there was some uncertainty about whether we wanted the dark prop to be listed at all. I think we have to list the dark prop on the React side if we want to do it this way.

Reply:
@jasperfurniss and @thestephenmarshall, I have fixed the Legend kit to be what I think Jason is suggesting. If you're both okay with the code, I will fix all the others to mimic this one, too.